consensus/polybft/consensus_runtime.go

```diff
@@ -120,12 +122,27 @@ func newConsensusRuntime(log hcf.Logger, config *runtimeConfig) *consensusRuntim
 	return runtime
 }
 
-// getLastBultBlockAndEpoch returns last build block and current epochMetadata in a thread-safe manner.
-func (c *consensusRuntime) getLastBultBlockAndEpoch() (*types.Header, *epochMetadata) {
+// getEpoch returns current epochMetadata in a thread-safe manner.
+func (c *consensusRuntime) getEpoch() *epochMetadata {
+	c.lock.RLock()
+	defer c.lock.RUnlock()
+
+	return c.epoch
+}
+
+// isCurrentEpoch checks if current epoch inside runtime is passed one
+func (c *consensusRuntime) isCurrentEpoch(epoch uint64) bool {
```

Comment:
I would revert this to the old behavior, since this method is only called on one place (`restartEpoch`), no need to extract this to the separate method.

Reply:
reverted to previous behavior https://github.com/0xPolygon/polygon-edge/commit/f0d08eb1a44c7b7b182c3f1757dc2e855bee7d57